src/components/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import HomePage from './homePage';
+
+const App = () => (
+    <div>
+        <Switch>
+            <Route exact path="/" component={HomePage} />
+        </Switch>
+    </div>
+)
+
+export default App;
```

Comment:
I have these components in my PR. They will not be required once it is merged.

Reply:
Yes, I rebase now yours are the one included.